RockWeb/Themes/NewSpring/assets/Lava/UI/locations-hero.lava

```diff
@@ -1,44 +1,43 @@
-{% assign campusSlug = 'Global' | PageParameter:'CampusSlug' | Replace:'-',' ' | Uppercase %}
-{% campus where:'Name == "{{ campusSlug }}"' iterator:'campuses' %}
-{% for campus in campuses %}
+{% assign campusSlug = 'Global' | PageParameter:'CampusSlug' | Replace:'-',' ' | Capitalize %}
+{% assign campuses = 'campuses' | PersistedDataset %}
+{% assign campus = campuses | Where:'Name', campusSlug | First %}
 
-    {% capture title %}{% if servicetype == 'Fuse' %}Fuse Student Ministry{% else %}{{ 'Global' | Attribute:'OrganizationName' }}{% endif %} in {{ campus.Name }}, SC | Locations | {% if servicetype == 'Fuse' %} Fuse | {% endif %}{{ 'Global' | Page:'SiteName' }}{% endcapture %}
+{% comment %}
+{% capture title %}{% if servicetype == 'Fuse' %}Fuse Student Ministry{% else %}{{ 'Global' | Attribute:'OrganizationName' }}{% endif %} in {{ campus.Name }}, SC | Locations | {% if servicetype == 'Fuse' %} Fuse | {% endif %}{{ 'Global' | Page:'SiteName' }}{% endcapture %}
+{% endcomment %}
+{% capture title %}{% if servicetype == 'Fuse' %}Fuse Student Ministry{% else %}{{ 'Global' | Attribute:'OrganizationName' }}{% endif %}{% if campus.Name != 'Eastlan' %} in {{ campus.Name }}, SC{% else %} - {{ campus.Name }}{% endif %} | Locations{% if servicetype == 'Fuse' %} | Fuse{% endif %} | {{ 'Global' | Attribute:'OrganizationName' }}{% endcapture %}
+{{ title | SetPageTitle:'PageTitle' }}
+{{ title | SetPageTitle:'BrowserTitle' }}
```

Comment:
Probably don't need both of these, if you leave off the parameter, it sets both I believe. Did we land on leaving this here and removing it from the block?
`{{ title | SetPageTitle }}`

Reply:
I don't think we ever talked about this at all. This part of the code actually didn't change. But I can for sure change it here if you think it makes sense to do so.

Reply:
I thought this was talking about this code? https://newspring.slack.com/archives/D04FDSWPC/p1585229590017800

Reply:
OH YEAH! I totally forgot about that. 🤦‍♂️

Yeah, I think we landed on putting this in the lava template and removing it from the block (because it wasn't really doing anything in the block anyway).